src/hooks/useImageProcessing.ts

```
@@ -124,5 +124,27 @@ export function useImageProcessing(eventId: string | null): UseImageProcessingRe
         }
     };
 
-    return { imageData, isUploading, error, uploadAndProcess, fetchImage };
-}
+    const logAccessibilityBypass = useCallback(
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)